pkg/controllers/cassandra/seedlabeller/control_test.go

```diff
@@ -63,23 +77,39 @@ func TestSeedLabellerSync(t *testing.T) {
 			navObjects:  []runtime.Object{cluster},
 			cluster:     cluster,
 			assertions: func(t *testing.T, state *controllers.State) {
-				CheckSeedLabel(pod0.Name, pod0.Namespace, t, state)
+				CheckSeedLabel(pod0.Name, seedprovider.SeedLabelValue, pod0.Namespace, t, state)
 			},
 		},
 		"add label if key missing": {
 			kubeObjects: []runtime.Object{ss0, pod0LabelMissing},
 			navObjects:  []runtime.Object{cluster},
 			cluster:     cluster,
 			assertions: func(t *testing.T, state *controllers.State) {
-				CheckSeedLabel(pod0.Name, pod0.Namespace, t, state)
+				CheckSeedLabel(pod0.Name, seedprovider.SeedLabelValue, pod0.Namespace, t, state)
 			},
 		},
 		"fix label if value incorrect": {
 			kubeObjects: []runtime.Object{ss0, pod0ValueIncorrect},
 			navObjects:  []runtime.Object{cluster},
 			cluster:     cluster,
 			assertions: func(t *testing.T, state *controllers.State) {
-				CheckSeedLabel(pod0.Name, pod0.Namespace, t, state)
+				CheckSeedLabel(pod0.Name, seedprovider.SeedLabelValue, pod0.Namespace, t, state)
+			},
+		},
+		"add multiple seeds": {
+			kubeObjects: []runtime.Object{ss0, pod0, pod1, pod2},
+			navObjects:  []runtime.Object{cluster},
+			cluster:     cluster,
+			assertions: func(t *testing.T, state *controllers.State) {
+				CheckSeedLabel(pod1.Name, seedprovider.SeedLabelValue, pod1.Namespace, t, state)
+			},
+		},
+		"don't add too many seeds": {
+			kubeObjects: []runtime.Object{ss0, pod0, pod1, pod2},
+			navObjects:  []runtime.Object{cluster},
+			cluster:     cluster,
+			assertions: func(t *testing.T, state *controllers.State) {
+				CheckSeedLabel(pod2.Name, "", pod2.Namespace, t, state)
```

Comment:
:question: Add a test for deleting label when the seeds value is reduced.

Reply:
Done